bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import datetime
+import logging
+from typing import Optional
+
+from discord import Color, Embed, Member
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    async def search_wikipedia(self, search_term: str) -> Optional[str]:
+        """Search wikipedia and return the first page found."""
+        async with self.http_session.get(SEARCH_API.format(search_term=search_term)) as response:
+            data = await response.json()
+        page = []
+
+        search_results = data["query"]["search"]
+        if len(search_results) == 0:
+            return None
+
+        # we dont like "may refere to" pages.
+        for search_result in search_results:
+            if "may refer to" in search_result["snippet"]:
+                pass
+            else:
+                page.append(search_result["title"])
+        log.info("appening titles")
+        return page
+
+    @commands.command(name="wikipedia", aliases=["wiki"])
+    async def w_pedia(self, ctx: commands.Context, *, search: str) -> None:
+        """Gives list of item."""
```

Comment:
This docstring does not really tell what this function is about, how it is used, I strongly suggest adding more information, otherwise invoking `.help wiki` will yield this

![image](https://user-images.githubusercontent.com/19921765/92308519-48142980-efc8-11ea-9275-697796c00d72.png)

A good docstring can yield good help as well, as seen here:
![image](https://user-images.githubusercontent.com/19921765/92308528-64b06180-efc8-11ea-8e04-98c2d73a7277.png)